ui/app/[locale]/(authorized)/kafka/[kafkaId]/nodes/DistributionChart.tsx

```diff
@@ -68,125 +90,205 @@ export function DistributionChart({
-              top: 30,
-            }}
-            width={width}
-          >
-            <ChartAxis
-              style={{
-                axis: { stroke: "transparent" },
-                ticks: { stroke: "transparent" },
-                tickLabels: { fill: "transparent" },
-              }}
-            />
-            <ChartStack>
-              {Object.entries(data).map(([node, data], idx) => (
-                <ChartBar
-                  key={idx}
-                  horizontal={true}
-                  barWidth={15}
-                  data={[
-                    {
-                      name: `Broker ${node}`,
-                      x: "x",
-                      y: getCount(data) ?? 0,
-                    },
-                  ]}
+    <Grid hasGutter>
+      <GridItem md={3}>
+        <Card>
+          <CardBody>
+            <DescriptionList isCompact isHorizontal>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  {t("DistributionChart.total_nodes")}{" "}
+                  <Tooltip content={t("DistributionChart.total_nodes_tooltip")}>
+                    <HelpIcon />
+                  </Tooltip>
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  {nodesCount.totalNodes}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  {t("DistributionChart.controller_role")}
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Icon status={"warning"}>
+                    <ExclamationTriangleIcon />
+                  </Icon>
+                  &nbsp; {nodesCount.totalControllers}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  {t("DistributionChart.broker_role")}
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Icon status={"success"}>
+                    <CheckCircleIcon />
+                  </Icon>
+                  &nbsp; {nodesCount.totalBrokers}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  {t("DistributionChart.lead_controller")}{" "}
+                  <Tooltip
+                    content={t("DistributionChart.lead_controller_tooltip")}
+                  >
+                    <HelpIcon />
+                  </Tooltip>
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Link href={`nodes/${leadController}`}>
+                    {t("DistributionChart.lead_controller_value", {
+                      leadController,
+                    })}
+                  </Link>
```

Comment:
Controller-only node configuration not retrievable - suggest to just drop the link.

```suggestion
                  {t("DistributionChart.lead_controller_value", {
                    leadController,
                  })}
```